src/main/java/ch/cern/eam/wshub/core/tools/Tools.java

```diff
@@ -303,4 +305,104 @@ public boolean isDatabaseConnectionConfigured() {
 		return (this.entityManagerFactory != null && getDataSource() != null);
 	}
 
+	public USERDEFINEDAREA getInforCustomFields(
+			InforContext context,
+			String previousClass,
+			USERDEFINEDAREA previousCustomFields,
+			String targetClass,
+			String entityType)
+			throws InforException {
+
+		/*	Table with all possible cases of inputs
+			IAE = IllegalArgumentException
+			prevC = previousClass
+			prevCFs = previousCustomFields
+			targetC = targetClass
+
+			+----+-------+---------+---------+----------------+---------------------------------+
+			| ID | prevC | prevCFs | targetC | description    | usage                           |
+			+----+-------+---------+---------+----------------+---------------------------------+
+			| 1  | null  | CF      | null    | prevCFs        | no previous class nor target    | [4]
+			| 2  | null  | CF      | ""      | merge with "*" | nullifying null class           | [3]
+			| 3  | null  | CF      | "D"     | merge with "D" | merge "*" into "D"              | [3]
+			| 4  | ""    | CF      | null    | IAE            | illegal argument                | [1]
```

Comment:
This is implementation specific. I would consider that previous class empty OR null means that there is no class set before. The distinction only happens on targetClass (specified vs not)